packages/frontend-shared/src/gql-components/Auth.vue

```diff
@@ -54,6 +75,7 @@ const isOnline = useOnline()
 
 const props = defineProps<{
   gql: AuthFragment,
+  errorType?: 'browserError' | 'loginError' | null,
```

Comment:
Can we make a type for this? It's used in a few places. I've just been making a `types.ts` file next to the component, so you could stick it in `gql-components/types.ts` or `gql-components/shared-types.ts`.

```ts
type AuthError = 'browserError' | 'loginError'
```

Also nit: could do away with `?`:

```ts
const props = withDefaults(defineProps<{
  errorType: AuthError | null
}>({
  errorType: null
})
```